letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  if integer > 89
+    letter_grade = "A"
+  elsif integer > 79
+    letter_grade = "B"
+  elsif integer > 69
+    letter_grade = "C"
+  elsif integer > 59
+    letter_grade = "D"
+  else
+    letter_grade = "F"
+  end
 
+  letter_grade
 end
 
 def shortest_string(array)
+  shortest = array.first
 
-  #Put your code here!
+  array.each do |string|
+    if string.length < shortest.length
```

Comment:
Check out [`Enumerable#min_by`](http://ruby-doc.org/core-2.4.2/Enumerable.html#method-i-min_by) for a shortcut on this.